core/src/main/java/io/questdb/cairo/vm/api/MemoryM.java

```diff
@@ -94,8 +94,4 @@ default void of(FilesFacade ff, LPSZ name, long extendSegmentSize, long size, in
     default void smallFile(FilesFacade ff, LPSZ name, int memoryTag) {
         of(ff, name, ff.getPageSize(), ff.length(name), memoryTag, CairoConfiguration.O_NONE, -1);
     }
-
-    default void wholeFile(FilesFacade ff, LPSZ name, int memoryTag) {
```

Comment:
We still have some readers/utilities that rely on `ff.length()` when mmapping a file: `TxReader#openTxnFile()`, `TableUtils#readTableName()`. Worth checking it.